demos/tutorial/intro.php

```diff
@@ -137,10 +137,13 @@ function init(): void {
 */
 
 session_start();
+
+$_SESSION['x'] = $_SESSION['x'] ?? [];
+
 $session = new \atk4\data\Persistence\Array_($_SESSION['x']);
 
 $form = \atk4\ui\Form::addTo($app);
-$form->setModel(new DemoInvoice($session))
+$form->setModel(new \atk4\ui\demo\DemoInvoice($session))
```

Comment:
should be not needed

Reply:
It doesn't work without, most probably because code is eval'ed

Reply:
I see, eval code without namespace, absolute path is probably better... Please check all other eval code.